File: app/src/main/java/com/ne1c/gitterclient/OnBootBroadcastReceiver.java
package com.ne1c.gitterclient;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.ne1c.gitterclient.Services.NewMessagesService;

public class OnBootBroadcastReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        context.startService(new Intent(context, NewMessagesService.class));
    }
}

File: app/src/main/kotlin/com/ne1c/gitteroid/services/OnBootBroadcastReceiver.kt
package com.ne1c.gitteroid.services

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class OnBootBroadcastReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        context.startService(Intent(context, NotificationService::class.java))
    }
}
